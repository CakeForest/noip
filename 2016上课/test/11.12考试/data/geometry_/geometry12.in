100
625 875 754 268 202 384 204 260 900 55 521 590 865 700 308 892 698 900 492 65 650 882 920 238 664 238 513 70 875 324 133 523 168 448 680 62 220 36 890 232 984 200 470 797 664 555 222 192 88 228 192 467 775 968 188 231 470 535 216 874 180 136 732 112 454 939 584 202 192 160 376 499 808 507 472 715 440 16 286 160 625 860 862 917 28 670 960 672 349 92 560 949 702 59 807 223 626 166 808 91 
525 370 278 64 170 750 632 838 210 737 150 386 39 900 817 72 500 300 32 52 89 120 290 116 557 248 570 269 222 780 261 32 924 622 490 152 276 500 720 548 672 324 200 88 452 744 539 410 794 863 215 786 144 752 880 490 530 911 394 362 568 800 139 26 444 732 636 950 128 804 102 672 192 760 540 853 158 38 676 100 416 388 431 436 925 757 62 0 120 720 112 335 255 368 340 630 218 500 352 25 
100
411 832
60 975
852 627
569 477
264 896
200 378
254 817
560 450
92 632
836 37
849 384
587 396
932 821
771 272
468 14
101 207
142 790
94 722
368 200
592 920
146 522
68 514
704 960
200 352
822 895
698 352
670 226
556 448
269 178
112 0
330 309
600 684
780 328
630 473
388 929
590 980
80 491
995 700
672 872
500 886
629 244
986 32
942 746
560 43
520 636
15 91
848 30
684 981
798 990
408 810
975 736
491 819
375 624
360 855
530 70
400 704
320 490
756 45
232 100
760 548
254 900
569 980
357 279
0 705
260 552
960 160
32 744
282 49
900 920
160 67
145 150
440 516
764 628
998 938
624 418
634 745
647 635
512 438
876 336
270 728
916 40
372 80
633 34
194 933
77 760
528 835
980 939
375 342
768 0
22 923
939 567
181 352
64 560
119 295
780 180
556 922
456 40
468 980
460 158
656 872
