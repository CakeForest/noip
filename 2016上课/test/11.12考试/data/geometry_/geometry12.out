100
100
100
100
100
65
100
100
76
88
100
100
100
100
51
40
99
90
62
100
73
69
100
62
100
100
93
100
51
12
69
100
100
100
100
100
67
100
100
100
91
100
100
62
100
14
88
100
100
100
100
100
100
100
62
100
85
79
41
100
100
100
69
76
86
100
85
41
100
32
39
99
100
100
100
100
100
99
100
100
97
48
69
100
91
100
100
76
77
99
100
61
69
50
99
100
53
100
65
100
